trade-finance-manager-api/src/v1/validation/validateIds.js

```diff
@@ -0,0 +1,60 @@
+const validator = require('validator');
+const teams = require('../../constants/teams');
+
+const isValidMongoId = (mongoId) => (mongoId ? validator.isMongoId(mongoId) : false);
+
+const isValidUkefNumericId = (ukefId) => {
+  const id = parseInt(ukefId, 10);
+
+  if (Number.isNaN(id)) {
+    return false;
+  }
+
+  const regex = /^\d{10}$/;
```

Comment:
Can all these REGEX please be defined and picked from CONSTANTS please.

Reply:
A single helper regex function will work where `isValidInput(REGEX_UKEFID, id)` and etc.